Library/Homebrew/extend/os/linux/system_config.rb

```diff
@@ -10,6 +10,9 @@ def host_os_version
         "#{description} (#{codename})"
       elsif (redhat_release = Pathname.new("/etc/redhat-release")).readable?
         redhat_release.read.chomp
+      elsif OS.cygwin?
```

Comment:
This should be in `extend/os/cygwin` instead.

Reply:
Fixed